_docs/readme.de.md

```diff
@@ -514,7 +517,7 @@ Konfiguration der Signaturen.
 - Eine Liste der IPv6-Signaturdateien, die CIDRAM zu analysieren versuchen sollte, getrennt durch Kommas. Hier können Sie Einträge hinzufügen, wenn Sie zusätzliche IPv6-Signaturdateien in CIDRAM aufnehmen möchten.
 
 ##### "block_cloud"
-- CIDRs blockieren, welche zu Web-/Server- Hostern gehören. Wenn sie eine API betreiben oder erwarten, dass sie oft Aufrufe von solchen Servern bekommen sollten sie diese option auf false (nicht blockieren) setzen. Wenn sie solche Anfragen blockieren möchen, setzen sie diese Option auf true [Standarteinstellung]
+- CIDRs blockieren, welche zu Web-/Server- Hostern gehören. Wenn sie eine API betreiben oder erwarten, dass sie oft Aufrufe von solchen Servern bekommen sollten sie diese Option auf false (nicht blockieren) setzen. Wenn sie solche Anfragen blockieren möchen, setzen sie diese Option auf true [Standarteinstellung].
```

Comment:
sie diese Option => Sie diese Option

Reply:
sie eine API => Sie eine API
sie oft Aufrufe => Sie oft Aufrufe
sie solche Anfragen => Sie solche Anfragen
sie diese Option => Sie diese Option

"sie" is the second form of you (the formal you, not the personal you - like you greet an unknown person or an important person with respect)

Reply:
Cheers. :+1:

Will fix now.